crates/router/src/connector/globalpay/transformers.rs

```diff
@@ -22,31 +23,212 @@ pub struct GlobalPayMeta {
 impl TryFrom<&types::PaymentsAuthorizeRouterData> for GlobalpayPaymentsRequest {
     type Error = error_stack::Report<errors::ConnectorError>;
     fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
-        let metadata: GlobalPayMeta = item.to_connector_meta()?;
-        let card = item.get_card()?;
-        let expiry_year = card.get_card_expiry_year_2_digit();
-        Ok(Self {
-            account_name: metadata.account_name,
-            amount: Some(item.request.amount.to_string()),
-            currency: item.request.currency.to_string(),
-            reference: item.attempt_id.to_string(),
-            country: item.get_billing_country()?,
-            capture_mode: item.request.capture_method.map(|f| match f {
-                enums::CaptureMethod::Manual => requests::CaptureMode::Later,
-                _ => requests::CaptureMode::Auto,
+        let metadata = item
```

Comment:
Why is to_connector_meta() is not used? Previously it was used for connector meta.

Reply:
I have changed to previous version